### ISSUE TITLE
Interior Rarity values don't affect rarity

### ISSUE THREAD

**Issue by swiftswordrush** [AudioKnight/Starlancer#36]
The values in both StarlancerWarehouse cfg and LethalLevelLoader cfg don't appear to change the rarity of the warehouse being chosen.  I set SLW cfg to 'list' with "experimentation:8000" and just in case set the LLL cfg file to "experimentation:8000".  In the debug window, the result says "Warehouse (50)".
[warehouse_cfg.txt](https://github.com/user-attachments/files/15749693/warehouse_cfg.txt)

**Comment by swiftswordrush** [comment]
Upon further reflection, I decided to check if this might be a problem with other mods causing conflicts.  So I created a new profile and loaded up only StarlancerWarehouse (and whatever mods are auto-associated to it, like LLL).  Opened the game modded so it would create the cfg files, and then within the LLL cfg I changed 'Manual Level Names List' to include "Experimentation:8000" and also changed 'Dynamic Level Tags List' to switch the all entry with "Experimentation:8000".  Debug window still shows Warehouse (50) in the Viable ExtendedDungeonFlows.
